lib/graph3d/Graph3d.js

```diff
@@ -392,8 +327,10 @@ Graph3d.prototype._checkValueField = function (data) {
     return;   // No need to check further
   }
 
+  var colValue = this.colValue;
```

Comment:
I would inline the value when the overhead of retrieving the value i.e. `this.colValue` is cheap. For expensive function calls etc. it would be indeed better to assign it once. Personal taste here.